eng/mgmt/automation/generation.yml

```diff
@@ -0,0 +1,61 @@
+trigger: none
+
+pr: none
+
+pool:
+  vmImage: 'ubuntu-18.04'
+
+variables:
+  MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
+  MAVEN_OPTS: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
+  NodeVersion: '14.x'
+
+steps:
+- task: Cache@2
+  inputs:
+    key: 'maven | "$(Agent.OS)" | **/pom.xml'
+    restoreKeys: |
+      maven | "$(Agent.OS)"
+      maven
+    path: $(MAVEN_CACHE_FOLDER)
+  displayName: Cache Maven local repo
+
+- bash: |
+    git config --global user.name "Azure-Fluent"
+    git config --global user.email "azfluent@microsoft.com"
+  displayName: Base Config
+
+- bash: |
+    sudo apt-get install -y --upgrade python3-pip python3-setuptools
+    pip3 install --upgrade wheel
+    pip3 install --upgrade PyYAML
+  displayName: Update python
+
+- task: NodeTool@0
+  displayName: 'Install Node.js $(NodeVersion)'
+  inputs:
+    versionSpec: '$(NodeVersion)'
+
+- bash: |
+    npm install -g autorest
+    npm install
+  displayName: 'Install autorest'
+
+- bash: |
+    export PATH=$JAVA_HOME_11_X64/bin:$PATH
+    java -version
+    ./eng/mgmt/automation/generate.py --readme "$(README)" --tag "$(TAG)" --service "$(SERVICE)" --version "$(VERSION)" --auto-commit-generated-code
+  displayName: Generation and Compile
+
+- bash: |
+    export MESSAGE="[Automation] Generate Fluent Lite from $(README)#$(TAG)"
+    export BRANCH="Fluent-lite-generation-$(Build.BuildId)"
+    git add -A
+    git commit -m "[Automation] Update other files outside generation folder"
+    git log -2
+    git push https://$(GITHUB_TOKEN)@github.com/azure-fluent/azure-sdk-for-java HEAD:refs/heads/$BRANCH -f
+    python3 eng/mgmt/pull-request.py chentanyi azure-sdk-for-java "$MESSAGE" "azure-fluent:$BRANCH"
+  displayName: Create pull request
+  env:
+    GITHUB_TOKEN: $(GITHUB_TOKEN)
```

Comment:
I don't think this is necessary to set this in env var given you are passing it as part of the URL.

Reply:
done